app/views/scorecards/_form.html.haml

```diff
@@ -0,0 +1,24 @@
+= form_for @scorecard, html: { multipart: true } do |f|

```

Comment:
Yup I sure am! This PR was just to change over to HAML syntax for my views instead of ERB